src/track/cue.h

```diff
@@ -17,21 +17,23 @@ class Cue : public QObject {
     Q_OBJECT
 
   public:
-    enum CueSource {
-        UNKNOWN   = 0,
-        AUTOMATIC = 1,
-        MANUAL    = 2,
+    enum class Source {
+        Unknown   = 0,
+        Automatic = 1,
+        Manual    = 2,
     };
 
-    enum CueType {
-        INVALID = 0,
-        CUE     = 1, // hot cue
-        LOAD    = 2, // the cue
-        BEAT    = 3,
-        LOOP    = 4,
-        JUMP    = 5,
-        INTRO   = 6,
-        OUTRO   = 7,
+    enum class Type {
+        Invalid = 0,
+        Hotcue  = 1,
+        MainCue = 2,
+        Beat    = 3, // unused (what is this for?)
+        Loop    = 4,
+        Jump    = 5,
+        Intro   = 6,
+        Outro   = 7,
+        FirstSound = 8, // not shown to user
```

Comment:
I'm still struggling with the naming. Thought about LeadIn/LeadOut, don't know if this would be even more misleading or confusing? You are the native speaker, so maybe I should just trust you ;)

Reply:
I think LeadIn/LeadOut would mean the same as Intro/Outro, but FirstSound/LastSound are not the same as Intro/Outro. They are placed at the same spot by AnalyzerSilence, but FirstSound/LastSound are unmovable and invisible to the user.